python/ray/autoscaler/aws/example-gpu-docker.yaml

```diff
@@ -9,16 +9,6 @@ min_workers: 0
 # node. This takes precedence over min_workers.
 max_workers: 2
 
-# The initial number of worker nodes to launch in addition to the head
```

Comment:
This looks like a breaking API change? We should avoid these for non experimental APIs.